testcases.py

```diff
@@ -379,6 +387,42 @@ def check(self):
       return False
     return self._check_files()
 
+
+class TestCaseHandshakeLoss(TestCase):
+  @staticmethod
+  def name():
+    return "handshakeloss"
+
+  @staticmethod
+  def testname():
+    return "multiconnect"
+
+  @staticmethod
+  def abbreviation():
+    return "L"
+
+  @staticmethod
+  def timeout() -> int:
+    return 300
```

Comment:
How long does this test usually take?

Reply:
I tried with quicly, and it's 150s or so IIRC